Cython/Utility/MemoryView_C.c

```diff
@@ -67,7 +67,7 @@ static CYTHON_INLINE $memviewslice_cname {{funcname}}(PyObject *obj, int writabl
 }
 
 
-/////////////// MemviewSliceValidateAndInit.proto ///////////////
+/////////////// MemviewSliceValidateAndInit.export ///////////////
```

Comment:
In the end, a fairly complex change for this little benefit. Let's see if we find more code to extract.